rosa_getting_started_sts/rosa-sts-deleting-access-cluster.adoc

```diff
@@ -1,12 +1,18 @@
 include::modules/attributes-openshift-dedicated.adoc[]
 [id="rosa-sts-deleting-access-cluster"]
-= Deleting access to a ROSA cluster
+= Revoking access to a ROSA cluster
 :context: rosa-sts-deleting-access-cluster
 
 toc::[]
 
-Delete access to a {product-title} (ROSA) cluster using the `rosa` command-line.
+Access to a {product-title} (ROSA) cluster is controlled through an identity provider (IDP). To remove access of a user to a cluster, you must configure that within the IDP that was set up for authentication.
```

Comment:
"through an identity provider"

Per the ISG:
> Through: Do not use to mean "by using".

I'm not quite sure of the context, but I would suggest revising this phrase. This sentence is also passive voice, so you may want to flip it around as well:

"An identity provider (IDP) controls access to a ROSA cluster"

Reply:
the context is to say that if you want to revoke a users access to the cluster, it must be done through the IdP that was set up.  Not sure if that helps better phrase the sentence.

Reply:
Thanks! Modified to this: _An identity provider (IDP) controls access to a Red Hat OpenShift Service on AWS (ROSA) cluster. To revoke access of a user to a cluster, you must configure that within the IDP that was set up for authentication._